### PR TITLE
GUACAMOLE-392: add basic travis-ci integration

### PR DESCRIPTION
**Pull request by chipitsine**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+install: true
+
+cache:
+  directories: [ $HOME/.m2 ]
+
+language: java
+
+script:
+  - mvn install -DskipTests=true -Drat.skip=true -Dmaven.javadoc.skip=true -B -V
```
Comment (mike-jumper) [inline .travis.yml]: As with apache/incubator-guacamole-server#110, I'm not confident Travis CI is a good choice. Depending on the result of discussion, if we do choose to go with Travis, skipping parts of the build shouldn't be done, especially tests and the RAT check. Catching test failures is one of the key points of using CI.

Comment (chipitsine) [reply .travis.yml]: as for skipping RAT check, I left that to those who understands how to deal with that :) I do not insist in skipping actually

**Comment by mike-jumper** [comment]
@chipitsine - as with apache/guacamole-server#110, please close this PR when you can. We are now using Jenkins for CI.

**Comment by chipitsine** [comment]
closing in favour of jenkins

**Pull request closed without merging**